scripts/ci/router-check.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
Is there a plan to drop this check form release repo and move to installer? Any reasons why? Or ca

Reply:
yes, my thought is that this is temporary, roughly mirrors this: https://github.com/openshift/release/blob/master/ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml#L86-#L152  @rabugopsl @alejovicu can work on moving these to a go test.

Reply:
> yes, my thought is that this is temporary

why do we need this in the first place?

Reply:
removed this script from installer repo